x-pack/plugins/security/server/audit/audit_events.ts

```diff
@@ -10,7 +10,7 @@ import type { EcsEventOutcome, EcsEventType, KibanaRequest, LogMeta } from 'src/
 import type { AuthenticationResult } from '../authentication/authentication_result';
 
 /**
- * Audit event schema using ECS format: https://www.elastic.co/guide/en/ecs/1.9/index.html
+ * Audit event schema using ECS format: https://www.elastic.co/guide/en/ecs/1.12/index.html
```

Comment:
@elastic/kibana-security Sorry for the codeowners ping for changing one line of commented code... but it was really gonna irritate me to not have this updated too 😛